content/en/docs/reference/score-spec-reference.md

```diff
@@ -91,7 +92,12 @@ resources:
     - **`Type`**: object.
     - **`Constraints`**: alphanumeric string key-value pairs.
 
-`resource-type`: specifies the resource in the target environment.
+`type`: specifies the resource type.
+
+- **Type**: string.
+- **Constraints**: alphanumeric string.
```

Comment:
alphanumeric with dashes

Reply:
Same for class below

Reply:
I assume the same is true for `resource-name` and `annotations`? https://docs.score.dev/docs/reference/score-spec-reference/#resources